packages/okta-angular/README.md

```diff
@@ -83,9 +83,38 @@ Routes are protected by the `OktaAuthGuard`, which verifies there is a valid `ac
   component: MyProtectedComponent,
   canActivate: [ OktaAuthGuard ]
 }
-``` 
+```
+
+By default, the `OktaAuthGuard` will redirect the user to Okta for authentication. You can modify this functionality by adding a `data` attribute:
+
+```typescript
+path: 'protected',
+component: MyProtectedComponent,
+canActivate: [ OktaAuthGuard ],
+data: {
+  authGuardRedirect: ({oktaAuth, router}) => {
+    // Redirect the user to a custom login page
+    router.navigate(['/custom-login-page']);
+  }
+}
+```
+
+Alternativly, set this behavior globally by adding it to your configuration object:
+
+```typescript
+const onAuthRequired = ({oktaAuth, router}) => {
+  // Do something else
+  router.navigate(['/custom-login-page']);
+}
+
+const config = {
```

Comment:
In these code snippets, we should rename `config` to something less generic (so that when you see it in an isolated context like this, you know what you're dealing with).  This value is currently typed as `OktaConfig` so we should call it `oktaConfig`